src/Proto.Cluster/Identity/IdentityStorageWorker.cs

```diff
@@ -131,9 +134,18 @@ private void RespondToWaitingRequests(IContext context, ClusterIdentity clusterI
                     else
                     {
                         //we have the lock, spawn and return
-                        (result, spawnLock) = await SpawnActivationAsync(activator, spawnLock, CancellationTokens.WithTimeout(5000));
+                        (result, spawnLock) = await SpawnActivationAsync(activator, spawnLock, CancellationTokens.WithTimeout(1000));
                     }
                 }
+                catch (OperationCanceledException e)
```

Comment:
Warning level logging for timeouts.
The system is not broken because we got a timeout in a retry loop